test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/FileBrowser/FileBrowserServiceTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
please sort the using statements per coding convention..System namespaces first alphabetically then other namespaces following alphabetically

Reply:
k